### DISCUSSION TITLE
String without label

### DISCUSSION THREAD

**Discussion by DidierMorandi** [WeblateOrg/weblate#11092] [General] [answered]
Hi there, working on a new translation project, I found thousand of strings tagged "NOT has:label".
How can a label be added, please?
Thanks.
(https://hosted.weblate.org/translate/godot-engine/godot-docs/fr/?q=NOT%20has:label)

**Comment by DidierMorandi** [comment, marked as answer]
I found this: https://docs.weblate.org/en/latest/admin/translating.html#additional
but my screen doesn't have Edit icons...

**Comment by NamelessGO** [comment]
Hi, is this about Godot or another project?

Only Project Maintainers can see "Edit" icons and change the String information

Would need to contact one of them, and ask to add labels:
https://hosted.weblate.org/projects/godot-engine/#information

![image](https://github.com/WeblateOrg/weblate/assets/66227691/b941a3eb-6414-4d6b-8c6f-24f4598d265a)

**Comment by DidierMorandi** [comment]
Yes, it is.
Problem is, there are 23 804 strings to "label": https://hosted.weblate.org/projects/godot-engine/godot-docs/fr/
I guess Projects Maintainers have something else to do, right?

**Reply by NamelessGO** [reply]
Yes, but it is not a major problem that requires fixing

labels and flags are there for organizing strings, not every project has to have them

Here is another example: Having an explanation and Screenshot helps a lot, but it is not like every string needs them
